src/main/kotlin/dev/cube1/minecord/plugin/listener/PlayerListener.kt

```diff
@@ -41,10 +42,30 @@ class PlayerListener : MinecordListener() {
         minecord.webhookClient.send(embed)
     }
 
+    /**
+     * Player Quit and Player Ban
+     */
     @EventHandler
     fun onPlayerQuit(event: PlayerQuitEvent) {
         discordChannel?.manager?.setTopic("서버 인원 : ${plugin.server.onlinePlayers.size-1}/${plugin.server.maxPlayers}")?.queue()
 
+        if (event.player.isBanned) {
```

Comment:
https://github.com/PaperMC/Paper/issues/9455

This code is calling an exception now, but it will be fixed.

Reply:
Fixed https://github.com/PaperMC/Paper/pull/9450